app/Http/Controllers/UserController.php

```diff
@@ -90,35 +91,99 @@ public function show()
 
     public function update(Request $request)
     {
-    	$this->validate($request, [
+        $this->validate($request, [
             'name' => 'required',
             'address' => 'required',
-            'phone' => 'required',
             'postcode' => 'required',
+            'city' => 'required',
             'state' => 'required',
             'country' => 'required',
-            'id' => 'required',
             'password' => 'required_if:change_password,true|confirmed',
         ]);
 
-    	$user = User::findOrFail(request()->id);
+        $user = auth()->user();
```

Comment:
This is not correct, as admin can change the other user profile, so it will required pass in user id to know which user to update profile